docs/src/modeler/data.md

```diff
@@ -17,11 +17,13 @@ The command line utility [jq](https://stedolan.github.io/jq/) offers even more
 features. The rest of this document provides example commands.
 
 ## View the entire file pretty-printed.
+
```

Comment:
Many of the examples on this page are out-of-date and incorrect.  I can update them.  However, I'm not positive that we want to keep the page.  What do you think?